packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeNodeRenderer.tsx

```diff
@@ -39,6 +39,8 @@ interface TreeNodeRendererOwnProps {
   actionButtonsClassName?: string;
   /** Tree node size. Should match the size passed to `TreeRenderer` component. */
   size?: "default" | "small";
+  /** Disables button-based filtering and hides filter buttons. Defaults to `false`*/
+  disableFiltering?: boolean;
```

Comment:
I think as suggested in the issue it would be better to use string literal union instead of boolean. Something like `filterButtonStyleType?: "show-on-hover" | "hide"`.

Reply:
was unsure about that, changed.